src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -237,6 +237,15 @@
   <data name="DoNotUseWeakCryptographicAlgorithmsMessage" xml:space="preserve">
     <value>{0} uses a weak cryptographic algorithm {1}</value>
   </data>
+  <data name="DoNotUseXslTransform" xml:space="preserve">
+    <value>Do Not Use XslTransform</value>
+  </data>
+  <data name="DoNotUseXslTransformDescription" xml:space="preserve">
+    <value>XslTransform may include dangerous external references.</value>
+  </data>
+  <data name="DoNotUseXslTransformMessage" xml:space="preserve">
+    <value>Construct XslTransform</value>
+  </data>
```

Comment:
Maybe you could just have the message and get rid of the description.
Do not use XslTransform.  It does not restrict potentially dangerous external references.